server/controllers.js

```diff
@@ -13,4 +14,18 @@ module.exports = {
       }
     });
   },
+
+  patch: (req, res) => {
+    const {id}= req.params;
+    const {is_favorite}=req.body;
+    models.patch(id, is_favorite, (err, result) => {
+      if (err) {
+        res.status(400);
+        res.send(err);
```

Comment:
If you want you can combine these into res.status(400).send(err).

Reply:
That is right. I will implement that in next PR.